defaults/main.yml

```diff
@@ -0,0 +1,108 @@
+---
```

Comment:
Since this defaults file is manually assembled, it's a huge maintenance burden. You could write a small script that parses the sympa man page and generates `defaults/main.yml`. We can then stage the script as well as the generated file and document against which version in which distribution we ran the script.

Reply:
I have read the whole config because its important that the settings make sense if you autogenerate it you can not check them